### PR TITLE
login and register ui

### PR DESCRIPTION
**Pull request by RoohiSingh-GH**

### CODE REVIEW

File: app/src/main/java/com/example/retailstudios/activities/LoginActivity.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import android.view.WindowManager
+import com.example.retailstudios.R
+import kotlinx.android.synthetic.main.activity_login.*
+
+class LoginActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+
+        @Suppress("DEPRECATION")
```
Comment (kunalgaba4) [inline app/src/main/java/com/example/retailstudios/activities/LoginActivity.kt:18]: Remove this code, if not neccessary.

File: app/src/main/java/com/example/retailstudios/activities/RegisterActivity.kt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+        @Suppress("DEPRECATION")
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            window.insetsController?.hide(WindowInsets.Type.statusBars())
+        } else {
+            window.setFlags(
+                WindowManager.LayoutParams.FLAG_FULLSCREEN,
+                WindowManager.LayoutParams.FLAG_FULLSCREEN
+            )
+        }
```
Comment (kunalgaba4) [inline app/src/main/java/com/example/retailstudios/activities/RegisterActivity.kt:25]: Remove the above code if not necessary.

File: app/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            android:layout_marginTop="16dp"
+            android:foreground="?attr/selectableItemBackground"
+            android:padding="@dimen/cickable_text_view_padding"
+            android:text="@string/register"
+            android:textColor="@color/colorSecondaryText"
+            android:textSize="@dimen/lbl_textview_textSize"/>
+    </LinearLayout>
+
+
+
```
Comment (kunalgaba4) [inline app/src/main/res/layout/activity_login.xml:136]: remove blank spavces

File: app/src/main/java/com/example/retailstudios/activities/RegisterActivity.kt [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        setContentView(R.layout.activity_register)
+
+        @Suppress("DEPRECATION")
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            window.insetsController?.hide(WindowInsets.Type.statusBars())
+        } else {
+            window.setFlags(
+                WindowManager.LayoutParams.FLAG_FULLSCREEN,
+                WindowManager.LayoutParams.FLAG_FULLSCREEN
+            )
```
Comment (kunalgaba4) [inline app/src/main/java/com/example/retailstudios/activities/RegisterActivity.kt:24]: remove this code if not neccessary.

File: app/src/main/java/com/example/retailstudios/activities/LoginActivity.kt (hunk 1 above)
Comment (RoohiSingh-GH) [reply app/src/main/java/com/example/retailstudios/activities/LoginActivity.kt:18]: This code here is for setting login activity to full screen in R android. This is necessary.

File: app/src/main/res/layout/activity_login.xml (hunk 3 above)
Comment (RoohiSingh-GH) [reply app/src/main/res/layout/activity_login.xml:136]: Will remove blank spaces, it was there just to visualize separately different components.

File: app/src/main/java/com/example/retailstudios/activities/RegisterActivity.kt (hunk 4 above)
Comment (RoohiSingh-GH) [reply app/src/main/java/com/example/retailstudios/activities/RegisterActivity.kt:24]: This whole code is to set the register page to full screen, not showing the status bar.

**Pull request merged**